scanner/cmd/scanner/main.go

```diff
@@ -103,19 +112,34 @@ func createGRPCService(backends *Backends) (grpc.API, error) {
 	if err != nil {
 		return nil, fmt.Errorf("identity extractor: %w", err)
 	}
-	// Create gRPC API service.
+	// Create gRPC API service and debug routes.
+	var customRoutes []routes.CustomRoute
+	for path, handler := range routes.DebugRoutes {
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         path,
+			Authorizer:    allow.Anonymous(),
+			ServerHandler: handler,
+			Compression:   true,
+		})
+	}
 	grpcSrv := grpc.NewAPI(grpc.Config{
```

Comment:
something I just realized, which is unrelated to this PR is we will definitely want to configure the server's keep-alive parameters: https://github.com/RTann/go-grpc-lb-poc/blob/main/cmd/server/server.go#L65

We should make the gRPC server configurable, as Scanner will want this, but the rest of StackRox probably won't

Reply:
Oh, yeah. It's OK to add this to Stackrox. Created https://issues.redhat.com/browse/ROX-18577